ambry-api/src/main/java/com/github/ambry/config/AccountStatsMySqlConfig.java

```diff
@@ -23,6 +27,8 @@ public class AccountStatsMySqlConfig {
   public static final String ENABLE_REWRITE_BATCHED_STATEMENT = PREFIX + "enable.rewrite.batched.statements";
   public static final String CONNECTION_IDLE_TIMEOUT = PREFIX + "connection.idle.timeout.ms";
   public static final String LOCAL_BACKUP_FILE_PATH = PREFIX + "local.backup.file.path";
+  public static final String ENABLE_CERTIFICATE_BASED_AUTHENTICATION =
+      PREFIX + "enable.certificate.based.authentication";
```

Comment:
we don't actually need this configuration for account stats and repair db, we can rely on the sslMode in the DBEnpoint.